src/main/java/cope/cosmos/client/features/modules/misc/FakePlayerModule.java

```diff
@@ -88,10 +99,56 @@ public void onEntityRemove(EntityWorldEvent.EntityRemoveEvent event) {
         }
     }
 
+    @Override
+    public void onUpdate() {
+        if (fakePlayer != null) {
+            Random random = new Random();
+            fakePlayer.moveForward = mc.player.moveForward + (random.nextInt(5) / 10F);
+            fakePlayer.moveStrafing = mc.player.moveStrafing + (random.nextInt(5) / 10F);
+            if (move.getValue()) travel(fakePlayer.moveStrafing, fakePlayer.moveVertical, fakePlayer.moveForward);
```

Comment:
Use separate lines (add brackets)

Reply:
Also, I'm assuming you can't (cannot check rn), but couldn't you call the travel method directly from the fake player (`fakePlayer.travel()`)?